static/js/typeahead_helper.js

```diff
@@ -79,14 +79,20 @@ var rendered = { persons: new Dict(), streams: new Dict(), user_groups: new Dict
 
 exports.render_person = function (person) {
     if (person.special_item_text) {
-        return exports.render_typeahead_item({ primary: person.special_item_text });
+        return exports.render_typeahead_item({
+            primary: person.special_item_text,
+            is_person: true,
+            img_src: 'static/images/megaphone.png',
+        });
     }
 
     var html = rendered.persons.get(person.user_id);
     if (html === undefined) {
         html = exports.render_typeahead_item({
             primary: person.full_name,
             secondary: person.email,
+            img_src: 'avatar/' + person.email,
```

Comment:
you should use the user ID here; both URLs work, but the email-based version is deprecated.

Reply:
@timabbott OK, I am now using the UID version, based on what is in `people.js`'s `small_avatar_url`. Please let me know if that was what you were referring to. Thanks!